<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="expInfoGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어_ac</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Exp Info</string>
   </property>
   <widget class="QLabel" name="nameLabel2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어 Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>InTaekJung</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="expTypeLabel2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어 Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pre-TEST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="expCntLabel2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어 Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1회차</string>
    </property>
   </widget>
   <widget class="QLabel" name="name2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이름: </string>
    </property>
   </widget>
   <widget class="QLabel" name="expCnt2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>회차:</string>
    </property>
   </widget>
   <widget class="QLabel" name="expType2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>유형:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TeInfoGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>771</width>
     <height>921</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어_ac</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Test Info</string>
   </property>
   <widget class="QGroupBox" name="testGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>641</width>
      <height>771</height>
     </rect>
    </property>
    <property name="title">
     <string>문제지</string>
    </property>
    <widget class="QLabel" name="testLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>621</width>
       <height>741</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ansGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>선택지</string>
    </property>
    <widget class="QRadioButton" name="ansRBtn1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="testCntLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>900</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1 / 6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="teSubmitBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>830</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제출하기</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="etcGBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>101</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>Etc.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
